ui/app/common/domain/services/programService.js

```diff
@@ -101,6 +101,16 @@ angular.module('bahmni.common.domain')
             return config ? config.showProgramStateInTimeline : false;
         };
 
+        var getEnrollmentInfoFor = function (patientUuid, representation) {
+            var params = {
+                patient: patientUuid,
+                v: representation
```

Comment:
Currently this method is called only from one place. Is there need to parameterize representation here? If yes can we construct params object from the place the function is called.(so that its consistent with others service calls)

Reply:
We are injecting the programService like other modules.